examples/gallery/3d_plots/scatter3d.py

```diff
@@ -58,7 +61,7 @@
     region=region,
     # Set frame parameters
     frame=[
-        "WsNeZ3",  # z axis label positioned on 3rd corner
+        'WsNeZ3+t"Iris flower data set"',  # z axis label positioned on 3rd corner, add title
         'xafg+l"Petal Width"',
         'yafg+l"Sepal Length"',
         'zafg+l"Petal Length"',
```

Comment:
Add units to axis labels.
```suggestion
        'xafg+l"Petal Width (cm)"',
        'yafg+l"Sepal Length (cm)"',
        'zafg+l"Petal Length (cm)"',
```